packages/zoe/src/contractFacet/types-ambient.d.ts

```diff
@@ -207,10 +207,7 @@ type ZcfSeatKit = {
   zcfSeat: ZCFSeat;
   userSeat: ERef<UserSeat>;
 };
-type HandleOffer<OR extends unknown, OA> = (
-  seat: ZCFSeat,
-  offerArgs?: OA,
-) => OR;
+type HandleOffer<OR extends unknown, OA> = (seat: ZCFSeat, offerArgs: OA) => OR;
```

Comment:
Not sure how this is related to the rest of the changes

Reply:
type bug discovered in the course of the work: d68596cb8d1d84db4c4eb9af754b132af1ce1649